extensions/geode-modules-session/build.gradle

```diff
@@ -18,25 +18,25 @@
 evaluationDependsOn(":geode-core")
 
 dependencies {
-  compile project(':extensions:geode-modules-session-internal')
-  compile project(':geode-core')
+  compile(project(':extensions:geode-modules-session-internal'))
+  compile(project(':geode-core'))
 
-  integrationTestCompile project(":geode-dunit")
-  integrationTestCompile ('org.httpunit:httpunit:' + project.'httpunit.version') {
+  integrationTestCompile(project(':geode-dunit'))
+  integrationTestCompile('org.httpunit:httpunit:' + project.'httpunit.version') {
     exclude group: 'javax.servlet'
     // this version of httpunit contains very outdated xercesImpl
     exclude group: 'xerces'
   }
   integrationTestCompile(group: 'org.eclipse.jetty', name: 'jetty-http', version: project.'jetty.version', classifier: 'tests')
   integrationTestCompile(group: 'org.eclipse.jetty', name: 'jetty-servlet', version: project.'jetty.version', classifier: 'tests')
-  integrationTestCompile 'javax.servlet:javax.servlet-api:' + project.'javax.servlet-api.version'
-  integrationTestCompile 'org.slf4j:slf4j-api:' + project.'slf4j-api.version'
+  integrationTestCompile('javax.servlet:javax.servlet-api:' + project.'javax.servlet-api.version')
```

Comment:
Is it outside the scope of this to make the dependencies use common syntax. Parameterized vs. concatenated string? +1 for Parameterized form like the lines above.

Reply:
Common syntax will be handled by the linter once we get it plumbed all the way through.